Codec/Picture/Extra.hs

```diff
@@ -130,3 +133,38 @@ rotateRight90 img@Image {..} =
   where
     gen x y = pixelAt img y (imageHeight - 1 - x)
 {-# INLINEABLE rotateRight90 #-}
+
+
+-- | Rotate image by 180, i.e flip both vertically and horizontally.
+
+rotate180 :: Pixel a => Image a -> Image a
```

Comment:
If we are to add rotation by 180°, it makes sense to have two versions: rotate to the left and rotate to the right, as they apparently produce not the same result.